client/src/Components/Header.js

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import AppBar from "@material-ui/core/AppBar";
+import Toolbar from "@material-ui/core/Toolbar";
+import IconButton from "@material-ui/core/IconButton";
+import Typography from "@material-ui/core/Typography";
+import SearchIcon from "@material-ui/icons/Search";
+import SettingsIcon from "@material-ui/icons/Settings";
+import TextField from "@material-ui/core/Input";
+import Grid from "@material-ui/core/Grid";
+
+const useStyles = makeStyles((theme) => ({
+  typography: {
+    fontFamily: theme.typography.fontFamily,
```

Comment:
You don't need this, actually. Because it's set on the theme.js, every typography will already have the fontFamily set there